x-pack/plugins/cases/public/components/actions/types.ts

```diff
@@ -13,6 +13,10 @@ export interface UseActionProps {
   isDisabled: boolean;
 }
 
+export interface UseCopyIDActionProps {
```

Comment:
nit: It seems more appropriate to put the type inside `useCopyIDAction` but this is a personal preference so its up to you 🙂

Reply:
I left it here to be consistent with the Props for the other actions 😅

Reply:
oh, I see. I missed that. Why not use the `UseActionProps` type? The plan was for all actions to have the same props.

Reply:
Well, I thought the same initially but then i noticed that `useSeverityAction` actually has different props.

In addition to that this action doesn't need `isDisabled`(not dependant on permissions) and doesn't need both `onAction` and `onActionSuccess`(for the other actions they are used to refresh cases and close the popover, we only need to close the Popover).

So instead of having unused params I created this new type. I could do something like `Pick<UseActionProps, onActionSuccess>` if that sounds better.

Reply:
Ok, that makes sense. I forgot about it. Maybe it is better if you `Pick` in case something changed in the future in the definition of the function.